recipe/migrations/cuda129.yaml

```diff
@@ -3,7 +3,7 @@ __migrator:
   kind:
     version
   migration_number:
-    1
```

Comment:
Looking for guidance on whether this migration_number should be incremented or not.

Reply:
Ok then don't bump the migration number.